test-apps/display-test-app/vite.config.ts

```diff
@@ -114,6 +115,30 @@ export default defineConfig(() => {
     define: {
       "process.env": process.env, // injects process.env into the frontend
     },
+    resolve: {
+      alias: {
+        "@itwin/appui-abstract": path.resolve(__dirname, '../../ui/appui-abstract/src/appui-abstract.ts'),
+        "@itwin/core-backend": path.resolve(__dirname, '../../core/backend/src/core-backend.ts'),
```

Comment:
this makes no sense, this is for bundling the frontend. i sure hope we arent pulling in the backend deps into our frontend